gluten-core/src/main/scala/org/apache/spark/sql/execution/ColumnarBroadcastExchangeExec.scala

```diff
@@ -76,16 +77,14 @@ case class ColumnarBroadcastExchangeExec(mode: BroadcastMode, child: SparkPlan)
           child,
           longMetric("numOutputRows"),
           longMetric("dataSize"))
-        val beforeCollect = System.nanoTime()
         val beforeBroadcast = System.nanoTime()
 
         longMetric("collectTime") += NANOSECONDS.toMillis(beforeBroadcast - beforeCollect)
+        longMetric("buildTime") += NANOSECONDS.toMillis(beforeBroadcast - beforeCollect)
```

Comment:
What is the expected difference between `collectTime` and `buildTime`? Looks like they have the same value.

Reply:
In Spark, `collectTime` is the elapsed time from child's execute and collect begin to end, `buildTime` is the time of BroadcastMode::transform.
In Gluten, ISparkPlanExecApi::createBroadcastRelation do the things, it is unnecessary to record `buildTime` actually, I will remove that.